Resources/Prototypes/_RMC14/Recipes/Materials/cardboard.yml

```diff
@@ -0,0 +1,654 @@
+# donut box
+- type: rmcConstruction
+  id: RMCDonutBoxBuild
+  name: donut box
+  prototype: RMCBoxDonutEmpty
+  noRotate: true
+  doAfterTime: 0.2
+  doAfterTimeMin: 0.2
+  materialCost: 1
+  ignoreBuildRestrictions: true
+  restrictedCollisionGroup: null
```

Comment:
All of these should inherit a prototype that has this information defined, this is repeated between all the prototypes
You can add a `parent: idhere` field in the YAML to do that